src/EditorFeatures/TestUtilities/Diagnostics/TestDiagnosticAnalyzerDriver.cs

```diff
@@ -14,7 +14,7 @@
 
 namespace Microsoft.CodeAnalysis.UnitTests.Diagnostics
 {
-    public class TestDiagnosticAnalyzerDriver : IDisposable
+    public class TestDiagnosticAnalyzerDriver
```

Comment:
This is a test only type though. If we want to preserve API for test types, I can revert it.  @jcouv @gafter?